docs/getting_started.md

```diff
@@ -56,6 +56,24 @@ Each `Admin::FooController` can be overwritten to specify custom behavior.
 Once you have Administrate installed,
 visit <http://localhost:3000/admin> to see your new dashboard in action.
 
+### Errors about assets?
+
+If your apps uses Sprockets 4, you'll need to add Administrate's assets to your `manifest.js` file. To do this, add these two lines to the file:
```

Comment:
Can we hard wrap this one like the rest of the file?

Reply:
Done!